server/tsuserver.py

```diff
@@ -221,6 +229,12 @@ def load_config(self):
                 'interval_length': 0,
                 'mute_length': 0
             }
+        if 'idle_timeout' not in self.config:
+            self.config['idle_timeout'] = {
+                'use_idle_timeout': False,
+                'kick_mods': False,
+                'length': 0
```

Comment:
I think that setting this to length of 0 you're allowing people to be kicked every 60 seconds if they don't have this configuration

Reply:
yeah, this is tolerable, since these three settings will always be defined by the user if the idle timeout is enabled